Extractor/Config.cs

```diff
@@ -642,6 +642,23 @@ public class SubscriptionConfig
         /// Log bad subscription datapoints
         /// </summary>
         public bool LogBadValues { get; set; } = true;
+
+        /// <summary>
+        /// Server namespace nodes the should be subscribed to a rebrowse upon changes to their values.
+        /// </summary>
+        public ServerNamespacesToRebrowseConfig? ServerNamespacesToRebrowse { get; set; }
```

Comment:
Maybe change the name in config? Since `Namespace` is already an OPC-UA concept, we don't really need to specify `ServerNamespace`.

Maybe we future proof it, and say something like `RebrowseTriggers`, or something?